README.md

```diff
@@ -34,8 +34,10 @@ UI components are located in [src/www/ui_components](src/www/ui_components). The
 
 Additional requirements for Android:
 
-* Android Studio 4+
-* Android SDK 29
+* [Android Studio 2020.3.1+](https://developer.android.com/studio)
+* [Latest Android Sdk Commandline Tools](https://developer.android.com/studio/command-line)
+* Android SDK 30 (with build-tools) via commandline `sdkmanager "platforms;android-30" "build-tools;30.0.3"`
```

Comment:
This is very helpful. But don't we need the NDK as specified at https://github.com/Jigsaw-Code/outline-client/blob/master/tools/build/setup_linux_android.sh?

Reply:
if they're starting from scratch, they'll need to add these as well - https://nisrulz.com/android-tips-tricks/android-aliases.html

Reply:
@fortuna From my experience I did not install NDK and build still succeeds.

@daniellacosse The environment variables (e.g., `ANDROID_HOME`) mentioned in the link are deprecated, I only need to add the following two:

```
export ANDROID_SDK_ROOT=$HOME/Android/Sdk
export PATH="$ANDROID_SDK_ROOT/cmdline-tools/latest/bin:$PATH"
```